draft-ietf-rtcweb-jsep.xml

```diff
@@ -1595,7 +1595,27 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               "max-bundle", and this is not the first m= section, or the BUNDLE
               policy is set to "balanced", and this is not the first m= section
               for this media type, an "a=bundle-only" line.</t>
-            </list></t>
+
+	      <t>If the RtpTransceiver's RtpSender is active, an "a=rid" line
+	      for each encoding specified by the application in the RtpSenders's
+	      parameters.  If no encodings have been specified, then one RID
+	      line is generated.  The rid line is specified in <xref
+	      target="I-D.ietf-mmusic-rid"></xref> and indicates the RID value
+	      used for sending media, as specified in <xref
+	      target="I-D.ietf-avtext-rid"></xref>.  The direction MUST be
+	      "send".  If the application has choosen a RID value, it MUST be
+	      used as the rid-identifier, otherwise a RID value MUST be chosen
+	      by the implemenatation.  Supported constraints MAY be included.
+	      </t>
+
+	      <t>If the RtpTransceiver's RtpSender is active and more than one
+	      "a=rid" line has been generated, an "a=simulcast" line, with
+	      direction "send" an d using the RID identification mechanism as
```

Comment:
It turns out that I don't even need to specify.  The latest draft only has one way (the RID way).